app/components/content-search/content-search.less

```diff
@@ -1,4 +1,4 @@
-custom-search-simple-content-search {
+ content-search {
```

Comment:
Moved styles here since they're basically the same and `custom-search-simple-content-search` now uses `content-search`.